aardvark/model.py

```diff
@@ -69,6 +69,9 @@ class AdvisorData(db.Model):
     @staticmethod
     def create_or_update(item_id, lastAuthenticated, serviceName, serviceNamespace, lastAuthenticatedEntity,
                          totalAuthenticatedEntities):
+        serviceName = serviceName[:128]
+        serviceNamespace = serviceNamespace[:64]
+        item = None
```

Comment:
this is required otherwise, when the next `try:` block fails, `item` is not defined and Aardvark will throw an exception when evaluating `if not item:` later